### ISSUE TITLE
TS search

### ISSUE THREAD

**Issue by isayev** [azag0/pyberny#4]
Kudos to the author, finally a descent open source python implementation! Quick question, can you do TS (saddle points) optimization out of the box?

**Comment by azag0** [comment]
No, it can't. Most of the ingredients are here though. One would need to (1) remove the [line interpolation](https://github.com/azag0/pyberny/blob/master/berny/berny.py#L67-L70) and (2) modify the quadratic step such that (ii) the second instead of the [first](https://github.com/azag0/pyberny/blob/master/berny/berny.py#L145) eigenvector of the RFO matrix is taken and (ii) the root of the shifted RFO eq is searched for between the first and second eigenvalue instead of [below the first](https://github.com/azag0/pyberny/blob/master/berny/berny.py#L153).

Also, while I'm sure that the algorithm used here is pretty close to the state of the art for minimization, I have no idea whether that would be also true for the TS search.

I'll leave this issue open as a feature request.

**Comment by espottesmith** [comment]
Hello.

Is this feature under active development, or are you still waiting for assistance? A native Python transition-state optimization algorithm would be greatly helpful for my research, and I would be interested in working to develop such an algorithm on top of the existing pyberny code.

**Comment by jhrmnn** [comment]
It is not under active development, and the status hasn't changed since my comment above.

I would most welcome this kind of effort and would be very happy to help if you encounter any issues or ambiguities with the existing code.

Also, I'm not aware that anyone else would be working towards this feature in Pyberny.

**Comment by espottesmith** [comment]
Excellent! I'll begin work and let you know what problems, if any, I encounter.
